fix CI errors

eddieluo72 opened the pull request:
Summary: fix CI error

Differential Revision: D60456623

dbort commented:
Example failure: https://github.com/pytorch/executorch/actions/runs/10269688919/job/28422892904?pr=4564#step:12:502

```
  Error looking up configured node root//runtime/executor:program (prelude//platforms:default#904931f735703749)

  Caused by:
      0: Error looking up configured node root//kernels/prim_ops:prim_ops_registry (prelude//platforms:default#904931f735703749)
      1: Error looking up configured node root//runtime/kernel:operator_registry (prelude//platforms:default#904931f735703749)
      2: Error resolving configuration deps of `root//runtime/kernel:operator_registry (prelude//platforms:default#904931f735703749)`
      3: Error getting configuration node of `shim//xplat/executorch/build/constraints:executorch-max-kernel-num-64` within the `prelude//platforms:default#904931f735703749` configuration
      4: Error running analysis for `shim//xplat/executorch/build/constraints:executorch-max-kernel-num-64 (<unbound>)`
      5: Error looking up configured node shim//xplat/executorch/build/constraints:executorch-max-kernel-num-64 (<unbound>)
      6: looking up unconfigured target node `shim//xplat/executorch/build/constraints:executorch-max-kernel-num-64`
      7: Error loading targets in package `shim//xplat/executorch/build/constraints` for target `shim//xplat/executorch/build/constraints:executorch-max-kernel-num-64`
      8: Error gathering package listing for `shim//xplat/executorch/build/constraints`
      9: Error listing dir `xplat/executorch/build/constraints`
     10: Error listing directory
     11: read_dir(/pytorch/executorch/shim/xplat/executorch/build/constraints)
     12: No such file or directory (os error 2)
```

eddieluo72 committed:
fix CI errors (#4564)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/4564

fix CI error

Differential Revision: D60456623
